docs/howto/how-to-build-compiler.md

```diff
@@ -208,3 +208,31 @@ NOTE: this assumes
 - host and target have same directoy structure
 - should copy `build` folder to target or
 - mounting `ONE` folder with NFS on the target would be simple
+
+## Known issues
+There's a potential known build error when attempting to cross-compile for ARM32 using GCC 10.5. You might encounter an error `comparison of unsigned expression in ‘< 0’ is always false [-Werror=type-limits]` reported from the `CircleNodeMixins.h` file. This is likely GCC's bug in this specific version. There's a workaround for it though - you can apply the following changes to both for loops in the `CircleNodeMixins.h`:
```

Comment:
can you please split lines within 100 cols for not h-scrolling?

Reply:
Sure, already done